src/middlewares/Conflict.js

```diff
@@ -44,6 +46,46 @@ class Conflict {
   static isTripRequestFound(req, res, next) {
     return DataEngine.findOne(req, res, next, tripRequests, { id: req.params.tripRequestId }, 'The trip request id not found');
   }
+
+  /**
+    * Check if trip exist
+    * @static
+    * @param {object} req request object
+    * @param {object} res response object
+    * @param {object} next next
+    * @memberof Conflict
+    * @returns {object} data
+    */
+  static isTripFound(req, res, next) {
+    return DataEngine.findOne(req, res, next, trips, { id: req.params.tripId }, 'The trip id not found');
+  }
+
+  /**
+* check is the request exist
```

Comment:
As a suggestion, we can specifically identify what this function is all about with JSdoc.
i.e: `check if room is booked`